src/assets/json/urbi-font.json

```diff
@@ -1 +1 @@
75c0-40.050-13.050-63-36.45-63-23.85 0-37.35 22.5-37.35 63zM153.6 378.753v-16.2c0-88.2 57.6-147.6 144-147.6 85.95 0 141.3 58.050 141.3 147.6v16.2c0 87.75-58.95 148.95-144 148.95s-141.3-59.4-141.3-148.95zM678.75 688.803v6.75c0 40.050 13.95 63.9 37.35 63.9 22.95 0 36.45-23.85 36.45-63.9v-6.75c0-40.050-13.050-63-36.45-63-23.85 0-37.35 22.5-37.35 63zM573.45 700.053v-16.2c0-88.2 57.6-147.6 144-147.6 85.95 0 141.3 58.050 141.3 147.6v16.2c0 87.75-58.95 148.95-144 148.95s-141.3-59.4-141.3-148.95zM250.35 778.803l227.7-286.2 200.25-261.9c24.75-32.4 54-30.6 74.7-15.3 11.25 8.1 18.45 18.9 19.35 31.5s-4.050 27.45-17.1 43.65l-199.35 251.1-228.6 296.55c-26.1 33.75-55.8 29.7-75.15 14.85-9-6.75-16.65-16.65-18.45-29.25-1.35-12.6 2.7-27.45 16.65-45z"],"attrs":[{}],"isMulticolor":false,"isMulticolor2":false,"grid":0,"tags":["offer-small"]},"attrs":[{}],"properties":{"order":1773,"id":1,"name":"offer-small","prevSize":32,"code":59701},"setIdx":0,"setId":1,"iconIdx":77},{"icon":{"paths":["M831.117 427.514c25.366 19.814 39.283 52.45 39.283 86.343v300.769c0 57.519-59.258 104.227-110.445 104.226l-516.017 2.748c-45.366 0.001-79.007-35.438-87.993-80.583-8.127-41.020 4.97-85.004 36.394-109.761l6.253-4.219 527.848-302.090c30.575-21.266 75.1-20.426 104.675 2.567zM759.682 816.452c-0.466 0 8.318-6.923 8.318-1.827v-295.478l-511.905 292.965c0.051 1.089-0.336 3.813-0.001 7.021l503.588-2.681zM299.9 566.2c-38.1 0-59.1-22.8-59.1-63.6v-271.8c0-55.5 20.1-77.1 71.4-77.1h107.7c94.2 0 152.7 56.1 152.7 144 0 94.8-60 144.9-159.9 144.9h-53.7v60c0 40.5-21.3 63.6-59.1 63.6zM359.3 357.4h37.8c39.3 0 59.4-15.3 59.4-57.6 0-36.3-19.8-57-56.1-57h-41.1v114.6z"],"attrs":[{}],"isMulticolor":false,"isMulticolor2":false,"grid":0,"tags":["raks-small"]},"attrs":[{}],"properties":{"order":1774,"id":0,"name":"racks-small","prevSize":32,"code":59702},"setIdx":0,"setId":1,"iconIdx":78},{"icon":{"paths":["M252.093 594.976c-26.666-2.087-35.106-37.063-12.326-51.081l431.063-265.269c20.85-12.83 46.853 5.785 41.427 29.657l-110.529 486.327c-6.025 26.511-42.691 29.298-52.655 4.003l-73.315-186.13-223.666-17.507z"],"attrs":[{}],"isMulticolor":false,"isMulticolor2":false,"grid":8,"tags":["location"]},"attrs":[{}],"properties":{"order":1714,"id":11,"name":"location","prevSize":24,"code":59655},"setIdx":0,"setId":1,"iconIdx":80}],"height":1024,"metadata":{"name":"urbi","url":"http://urbi.co"},"preferences":{"showGlyphs":true,"showQuickUse":true,"showQuickUse2":true,"showSVGs":true,"fontPref":{"prefix":"icon-","metadata":{"fontFamily":"urbi","majorVersion":1,"minorVersion":1,"fontURL":"http://urbi.co"},"metrics":{"emSize":1024,"baseline":6.15,"whitespace":50},"embed":false,"includeMetadata":false,"showSelector":true,"showMetadata":true,"showVersion":true,"showMetrics":true,"selector":""},"imagePref":{"prefix":"icon-","png":true,"useClassSelector":true,"color":0,"bgColor":16777215,"classSelector":".icon","name":"icomoon","height":32,"columns":16,"margin":16},"historySize":50,"showCodes":false,"gridSize":16,"showLiga":false}}
```

Comment:
if more icons were added, there should probably be a change in the `urbi.ttf` file included in the showcase app, too. Unless that didn't change? Sounds odd, though

Reply:
Thanks. Commits 9faab52 https://github.com/urbi-mobility/urbi-react-native/pull/233/commits/37fd5abad6c184df3730d704255e0b1cb7cc322b